front/production_checks/temporal/workflows.ts

```diff
@@ -0,0 +1,12 @@
+import { proxyActivities } from "@temporalio/workflow";
+
+import type * as activities from "@app/production_checks/temporal/activities";
+
+const { runAllChecksActivity } = proxyActivities<typeof activities>({
+  startToCloseTimeout: "1 hour",
+  heartbeatTimeout: "5 minutes",
```

Comment:
Are we heartbeating in between checks? this will break fast otherwise?

Reply:
yes correct.